build.sbt

```diff
@@ -88,7 +88,10 @@ val k8s = Seq(
   "io.kubernetes" % "client-java" % "2.0.0" excludeAll ExclusionRule(organization = "org.bouncycastle")
 ) ++ bouncyCastle
 
-val natslibs = Seq("io.nats" % "java-nats-streaming" % "2.1.2")
+val natslibs = Seq(
+  "io.nats" % "java-nats-streaming" % "2.1.2" exclude("io.nats", "jnats"),
+  "io.nats" % "jnats" % "2.1.0"
```

Comment:
Was there a dependency problem?

Reply:
Yes, in the new version of jnats they changed the configuration builder so that when you set username and password, they are converted to char arrays. Currently, we are passing empty username and password as nulls, so in the new version of jnats, `.toCharArray()` method throws null pointer exception...

Reply:
thanks, good to know.